GitUI/UserControls/RevisionGrid/CellStyle.cs

```diff
@@ -6,17 +6,19 @@ namespace GitUI.UserControls.RevisionGrid
     {
         public readonly Brush BackBrush;
         public readonly Color ForeColor;
+        public readonly Color GrayForeColor;
         public readonly Font NormalFont;
         public readonly Font BoldFont;
         public readonly Font MonospaceFont;
 
-        public CellStyle(Brush backBrush, Color foreColor, Font normalFont, Font boldFont, Font monospaceFont)
+        public CellStyle(Brush backBrush, Color foreColor, Color grayForeColor, Font normalFont, Font boldFont, Font monospaceFont)
```

Comment:
I don't think it is a good idea to mention colour in the name of a variable. Colours do change, and one day (on in another theme) `grayForeColor` may as well be blue or white...
This could named as "secondaryForeColor" or "commitBodyForeColor", or something along these lines.

Reply:
They use `SystemColors.GrayText` though. I personally would prefer `SystemColors.DisabledText`, but we have what we have, and introducing a different name for disabled color would confuse things IMO.